README.md

```diff
@@ -1,35 +1,351 @@
 
 ## Additional Configuration
-Aside from which detectors to scan on, you can add additional fields to your config, `./nightfall/config.json`, to ignore tokens and files as well as specify which files to exclusively scan on.
+
+Aside from which conditions to scan on, you can add additional fields to your config, `./nightfall/config.json`, to ignore tokens and files as well as specify which files to exclusively scan on.
+
 ### Token Exclusion
+
 To ignore specific tokens, you can add the `tokenExclusionList` field to your config. The `tokenExclusionList` is a list of strings, and it mutes findings that match any of the given regex patterns.
 
 Here's an example use case:
 
-```tokenExclusionList: ["NF-gGpblN9cXW2ENcDBapUNaw3bPZMgcABs", "^127\\."]```
+`tokenExclusionList: ["NF-gGpblN9cXW2ENcDBapUNaw3bPZMgcABs", "^127\\."]`
 
 In the example above, findings with the API token `NF-gGpblN9cXW2ENcDBapUNaw3bPZMgcABs` as well as local IP addresses starting with `127.` would not be reported. For more information on how we match tokens, take a look at [regexp](https://golang.org/pkg/regexp/).
+
 ### File Inclusion/Exclusion
+
 To omit files from being scanned, you can add the `fileExclusionList` field to your config. In addition, to only scan on certain files, add the `fileInclusionList` to the config.
 
 Here's an example use case:
+
 ```
+
     fileExclusionList: ["*/tests/*"],
     fileInclusionList: ["*.go", "*.json"]
+
 ```
+
 In the example, we are ignoring all file paths with a `tests` subdirectory, and only scanning on `go` and `json` files.
 Note: we are using [gobwas/glob](https://github.com/gobwas/glob) to match file path patterns. Unlike the token regex matching, file paths must be completely matched by the given pattern. e.g. If `tests` is a subdirectory, it will not be matched by `tests/*`, which is only a partial match.
+
+## Extra Real Examples
+
+To summarize, we provide sevaral more examples as below
+
+- Config conditionSet through app.nightfall.ai UI
+
+```json
+{ "conditionSetUUID": "UUID HERE" }
+```
+
+- Config conditions inline with prebuilt detector
+
+```json
+{
+  "conditions": [
+    {
+      "minNumFindings": 1,
+      "minConfidence": "POSSIBLE",
+      "detector": {
+        "detectorType": "NIGHTFALL_DETECTOR",
+        "nightfallDetector": "API_KEY",
+        "displayName": "nfAPIKEY"
+      }
+    },
+
+    {
+      "minNumFindings": 2,
+      "minConfidence": "POSSIBLE",
+      "detector": {
+        "detectorType": "NIGHTFALL_DETECTOR",
+        "nightfallDetector": "CREDIT_CARD_NUMBER",
+        "displayName": "nfCC"
+      }
+    }
+  ]
+}
+```
+
+- Config conditions inline with your own regex | word list
+
+```json
+{
+  "conditions": [
+    {
+      "minNumFindings": 1,
+      "minConfidence": "POSSIBLE",
+      "detector": {
+        "detectorType": "REGEX",
+        "regex": {
+          "pattern": "coupon",
```

Comment:
same here - let's use a slightly more complex regex instead of just a word

Reply:
same as above, added coupon:\\d{4,}